exercises/hello-world/src/HelloWorld.purs

```diff
@@ -1,6 +1 @@
 module HelloWorld where
-
```

Comment:
I looked at the tracks for Haskell and Elm and saw that they didn't provide the typesig so I stopped providing them too. HelloWorld was made before I realised this so I'm removing it to match the others.